dashboard/charts.py

```
@@ -8,12 +8,10 @@
 """
 
 from fasthtml.common import Div, P
-from monsterui.all import Card, DivLAligned, Loading, LoadingT, TextPresets
+from monsterui.all import Card, DivLAligned, Loading, LoadingT, TextPresets, ApexChart
 import pandas as pd
-import plotly.graph_objects as go
-from plotly.subplots import make_subplots
-
-from dashboard.app import app
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)